type-generator/src/main/java/io/mesosphere/types/dtr/compiler/internal/BlockArguments.java

```diff
@@ -0,0 +1,25 @@
+package io.mesosphere.types.dtr.compiler.internal;
+
+import java.util.Map;
+
+/**
+ * A set of parameters that are passed down to a code block,
+ * defining the template fields and scope.
+ */
+public interface BlockArguments {
```

Comment:
Why is this empty?

Reply:
The idea of the `BlockArgument` is to annotate whatever can be passed as an argument to the compiler.